docker-compose.yml

```diff
@@ -1,10 +1,162 @@
+# This docker-compose file will stand up the entire stack including ingress and inventory.
 version: "3"
 services:
+  zookeeper:
+    image: confluentinc/cp-zookeeper
+    environment:
+      - ZOOKEEPER_CLIENT_PORT=32181
+      - ZOOKEEPER_SERVER_ID=1
+      - KAFKA_OPTS='-Dzookeeper.admin.enableServer=false'
```

Comment:
I had to remove the quotes, otherwise the variable will keep the quotes (at least if using `podman-compose`)